packages/components/psammead-episode-list/src/Episode.jsx

```diff
@@ -8,22 +8,24 @@ import {
 } from '@bbc/gel-foundations/breakpoints';
 import tail from 'ramda/src/tail';
 import pathOr from 'ramda/src/pathOr';
-import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
 import MediaIndicator from './MediaIndicator';
 import Image from './Image';
 
 const Wrapper = styled.div`
   position: relative;
+  &:focus,
```

Comment:
Maybe I'm misunderstanding the basics here, but in what scenario will this `div` be focused?

Wouldn't we have to give it a tab-index, which seems undesirable because we wouldn't want a non-interactive div in the tab order

Reply:
oh you're right. I was testing by adding focus in the dev tools. needs to go on the anchor

Reply:
Aye - the hover was added to this div because there was a requirement that the SVG had to be outside the link (yet still respond to hovers)

I believe that requirement is relaxed now and the SVG can go inside the link, so it should be possible for both the `:hover` and `:focus` to be on the `a`

Reply:
Cool. I'll fix this in the morning